0. Kirish/0.2 Tamoyillari.md

```diff
@@ -0,0 +1,32 @@
+# 0.2 Tamoyillari
+Ushbu qismda biz Me'yor tamoyillarini o'rganamiz. Ushbu tamoyillar me'yorga mos keladigan tizimning barcha komponentlariga taalluqlidir. Xo'sh ushbu komponentlar brokerlar, xizmatlar yoki taqdim etuvchilar bo'ladimi?
+
+## 0.2.0 Birinchi navbatda - Odamlar
+Ushbu tamoyilning asosiy g'oyasi odamlarni hisobga olgan holda me'yorga mos keladigan tizimlarni qurish va loyihalashdir. Nafaqat tizimdan foydalanmoqchi bo'lganlar, balki uni saqlab qoladigan va rivojlantiradigan odamlarni ham hisobga olgan holda me'yorga mos keladigan tizimlarni qurish va loyihalashdir.
+
+Tizim "Birinchi navbatda - Odamlar" tamoyiliga amal qilishi uchun u murakkablikdan ko'ra oddiylikni hurmat qilishi kerak. Oddiylik qayta yozish imkoniyatini beradi. Va, shuningdek, U tizimning umumiy namunasida haqiqiy murakkablilikni ta'minlash uchun aqliy fikrlash bilan qurilgan o'zgartirib bo'lmas tizimlarni loyihalash imkoyatini ham beradi.
```

Comment:
`It also leads to designing monolithic systems built with a modular mindset to allow a true fractality in the overall pattern of the system.` => bu gap ozgina dastur arxitekturasi ilmidan xabardor bo'lishni talab qiladi. 

Sizning tarijangiz: 
`Va, shuningdek, U tizimning umumiy namunasida haqiqiy murakkablilikni ta'minlash uchun aqliy fikrlash bilan qurilgan o'zgartirib bo'lmas tizimlarni loyihalash imkoyatini ham beradi.`

Meni taklifim: 
`Bu monolit (yakka-asosli) sistemalarni umumiy andazasida haqiqiy bo'laklana olish xususiyati berish uchun xayolan mayda parchalarga bo'lib loyihalashni taqazo qiladi`